### PR TITLE
add endpoints

### PR DESCRIPTION
**Pull request by charlie-kano**

### CODE REVIEW

**Review by onishiweb — APPROVED** [review]

File: lib/admin.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ const ENDPOINTS = {
     getFlaggedComments(offset) {
         return `/flagged-comments${offset ? `?offset=${offset}` : ''}`;
     },
+    setFlaggedShareReviewed(shareId) {
+        return {
+            path: `/flagged-shares/${shareId}/reviewed`,
+            method: 'PATCH',
+            responseType: 'text'
```
Comment (AliceMarshall) [inline lib/admin.js:25]: are you sure this is the response type?

Comment (AliceMarshall) [reply lib/admin.js:25]: yeh this shouldn't have a response type it's just the 200 status you get returned

File: lib/admin.js [hunk 2]
```diff
@@ -18,6 +18,20 @@ const ENDPOINTS = {
...
+        }
+    },
+    setFlaggedCommentReviewed(commentId) {
+        return {
+            path: `/flagged-comments/${commentId}/reviewed`,
+            method: 'PATCH',
+            responseType: 'text'
```
Comment (AliceMarshall) [inline lib/admin.js:32]: for this too also

File: lib/admin.js (hunk 1 above)
Comment (charlie-kano) [reply lib/admin.js:25]: the response is empty. leaving it as default was throwing an error in the response data processing (as it was trying to parse JSON). this is how other empty responses are handled so added it

Comment (AliceMarshall) [reply lib/admin.js:25]: hmmmmmmmmm v odd - kk do your thing 👍 ✨

**Pull request merged**